ts/editable/ContentEditable.svelte

```diff
@@ -61,6 +61,9 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         &:focus {
             outline: none;
         }
+        & :global(pre) {
+            white-space: pre-wrap;
+        }
     }
 
     /* editable-base.scss contains styling targeting user HTML */
```

Comment:
As I've mentioned here, you should put css targeting the user's input into editable-base.scss

Reply:
Ah yes, thanks. I've searched for something like this but somehow overlooked it ^^